### ISSUE TITLE
DBI 598 unable to use USB 3.0 in MTP Responder

### ISSUE THREAD

**Issue by kraussian** [rashevskyv/dbi#253]
Despite enabling USB 3.0 in Atmosphere (`usb30_force_enabled = u8!0x1`) and connecting to a USB-C 3.2 Gen 2 port on PC, DBI is showing that the actual speed used is USB 2.0 instead of 3.0.

![2023072909114000-CC47F0DEC75C1FD3B1F95FA9F9D57667](https://github.com/rashevskyv/dbi/assets/29044865/904f01ee-e255-4182-ac0f-e9202ff6fb28)

**Comment by kraussian** [comment]
> Что за кабель используется? Какая модель приставки? Switch lite не имеет USB 3

I'm using Switch OLED, and have tried multiple cables which support USB 3.0 when connected with my external SSD.

**Comment by muxi1** [comment]
@kraussian USB 3.0 support works if the cable supports USB 3.0. If it still does not work, simply reverse the plug on the UBC-C port.

DBI_628

**Comment by rashevskyv** [comment]
On older dbi versions usb 3 detection wasn't works correctly

ср, 8 лист. 2023 р., 11:08 користувач muxi1 ***@***.***> пише:

**Comment by muxi1** [comment]
@rashevskyv Yeah, you are right. I had been using a USB cable that does not support USB 3.0 the whole time. Sorry for the inconvenience.

**Comment by Evennnet** [comment]
I encountered an issue with Windows 10/11 where USB-C to C connections were not being recognized. I have two hacked consoles; one could successfully connect via Type-C, while the other could not. I tried overlaying both consoles with identically configured files but to no avail. Ultimately, rebuilding the eMMC NAND was the solution that allowed Windows 10/11 to successfully recognize and transfer files via Type-C.
